sync/src/com/kirinpatel/net/Media.java

```diff
@@ -0,0 +1,100 @@
+package com.kirinpatel.net;
+
+import java.io.Serializable;
+import java.nio.file.Path;
+
+public class Media implements Serializable {
+
+    private String url;
+    private String filePath;

```

Comment:
Was not possible as the User class implements a Media object. The connected users are sent from the server to clients and still cause the exception.